views/download.handlebars

```diff
@@ -2,13 +2,13 @@
   <script src="/download.js"></script>
   <div id="download-page-one">
     <div class="title">
-      <span id="dl-filename"
+      <span id="dl-file"
+            data-filename="{{filename}}"
+            data-size="{{sizeInBytes}}"
+            data-ttl="{{ttl}}"
             data-l10n-id="downloadFileName"
             data-l10n-args='{"filename": "{{filename}}"}'></span>
```

Comment:
Unrelated to this glorious PR, but looking at it with my special eyes, I think this is a bug...
I can seemingly break the download page hard on production by uploading a file with a pair of double quotes in the filename. Not sure if we can encode the filename here (and on line 6 above, and possibly elsewhere, or what.